prow/plugins/require-matching-label/require-matching-label.go

```diff
@@ -244,3 +256,49 @@ func handle(log *logrus.Entry, ghc githubClient, cp commentPruner, configs []plu
 	}
 	return nil
 }
+
+func handleCommentEvent(pc plugins.Agent, ce github.GenericCommentEvent) error {
+	// Only consider open PRs and new comments.
+	if ce.IssueState != "open" || ce.Action != github.GenericCommentActionCreated {
+		return nil
+	}
+	// Only consider "/check-require-labels" comments.
+	if !checkRequireLabelsRe.MatchString(ce.Body) {
+		return nil
+	}
+
+	cp, err := pc.CommentPruner()
+	if err != nil {
+		return err
+	}
+
+	return handleComment(pc.Logger, pc.GitHubClient, cp, pc.PluginConfig.RequireMatchingLabel, &ce)
+}
+
+func handleComment(log *logrus.Entry, ghc githubClient, cp commentPruner, configs []plugins.RequireMatchingLabel, e *github.GenericCommentEvent) error {
+	org := e.Repo.Owner.Login
+	repo := e.Repo.Name
+	number := e.Number
+
+	event := &event{
+		org:    org,
+		repo:   repo,
+		number: number,
+		author: e.User.Login,
+	}
+	if e.IsPR {
```

Comment:
The `handle()` logic already knows how to pull labels down - why do we need this here?

Reply:
My bad, I've simplified the function to omit adding currentLabels field